setup.py

```diff
@@ -27,5 +123,6 @@ def extract_requirements(filename):
         'console_scripts': [
             'tendrl-node-agent = tendrl.node_agent.manager.manager:main'
         ]
-    }
+    },
+    cmdclass={'bump': BumpCommand, 'release': ReleaseCommand},
```

Comment:
I think `bump` is faster to type, but I've changed it to `bumpversion` now so we can continue :)